doc/man1/openssl-x509.pod.in

```diff
@@ -506,13 +506,14 @@ F<.srl> appended. For example if the CA certificate file is called
 F<mycacert.pem> it expects to find a serial number file called
 F<mycacert.srl>.
 
+If the B<-CA> option is specified and both the <-CAserial> and <-CAcreateserial>
+options are not given and the default serial number file does not exist,
+a random number is generated; this is the recommended practice.
+
 =item B<-CAcreateserial>
 
 With this option the CA serial number file is created if it does not exist:
-it will contain the serial number "02" and the certificate being signed will
-have the 1 as its serial number. If the B<-CA> option is specified
-and the serial number file does not exist a random number is generated;
-this is the recommended practice.
+a random number is generated and stored; this is the recommended practice.
```

Comment:
I still think this does not make any sense. Why it was talking about the serial "02" before? Wasn't it really meant to mean that if `-CAcreateserial` is present, the serial number file will be created and it will start from `02` in it and the cert will have `1` as the serial number. Where if the _implicit_ (from `-CAfile`) serial number file does NOT exist (and `-CAserial` is NOT used) a random serial number is being used - you did this part right now.

Reply:
But yeah, as this was the behavior already, maybe just document that. So something like: `With this option the CA serial number file is created if it does not exist. A random number is generated, used for the certificate, and saved into the serial number file in that case.`

Reply:
Apparently, some time before 1.1.1, on creation of a serial number file (and likely independent of the file name being given or derived by default), the initial contents of the file were like this.
Looks like this was changed at some time, but  the doc got not updated.

Anyway, now, whenever the serial number is not given in any way, it is uniformly created as a random number. And the doc now correctly reflects this.

Reply:
> But yeah, as this was the behavior already, maybe just document that. So something like: `With this option the CA serial number file is created if it does not exist. A random number is generated, used for the certificate, and saved into the serial number file in that case.`

I improved the wording of the new sentence as you suggested.